I.write-tests/03-remove-vowels-from-array/remove-vowels-in-array.test.js

```diff
@@ -1,10 +1,17 @@
+var removeVowelsForWords = require('./remove-vowels-in-array');
+
 test('remove vowels from all words in array', function () {
     // Arrange
-
+    const expected = ["rn", "tz", "dnl"]
     // Act
 
+    var words = ["Irina", "Etza", "Daniel", "clement"];
```

Comment:
if we add `"Mikey"` to the array, we'd expect the result to include `"mky"`
Can we test it and see if we can spot any bug in the code? ;)